extensions/amp-orientation-observer/validator-amp-orientation-observer.protoascii

```diff
@@ -49,6 +49,10 @@ tags: {  # <amp-orientation-observer>
     # Values such as: "100 100", "0 100", etc..
     value_regex: "(\\d+)\\s{1}(\\d+)"
   }
+  attrs: {
+    name: "smoothing"
+    value_regex: "([0-9]+)?"
```

Comment:
warning: I'm not sure I understand what you are trying to accomplish.

Would `[0-9]+` work?

value_regex is always full-match syntax in the validator for security, ex `^{value_regex}$`.

Reply:
Hey @gregable  actually need help with this, it's a number that also has a default so it's also possible to just include the attribute without a value, not sure if that's a good pattern

Reply:
I believe it would be `[0-9]+|` then since it's a full match, which means it'll be evaluated as `^[0-9]+|$` which means it can be at least one digit or nothing. Could add some tests to verify.